redirects-list.json

```diff
@@ -37,6 +37,10 @@
   "^/doc/get-started(/.*)?$                                                               /doc/start",
   "^/doc/start/data-versioning$                                                           /doc/start/data-and-model-versioning",
   "^/doc/start/data-access$                                                               /doc/start/data-and-model-access",
+  "^/doc/start/data-and-model-versioning(/.*)?$                                           /doc/start/data$1",
+  "^/doc/start/data-and-model-access(/.*)?$                                               /doc/start/data/access$1",
+  "^/doc/start/data-pipelines(/.*)?$                                                      /doc/start/data/pipelines$1",
+  "^/doc/start/metrics-parameters-plots(/.*)?$                                            /doc/start/data/metrics-parameters-plots$1",
```

Comment:
```suggestion
  "^/doc/start/data-and-model-versioning(/.*)?$                                           /doc/start/data-management$1",
  "^/doc/start/data-and-model-access(/.*)?$                                               /doc/start/data-management/access$1",
  "^/doc/start/data-pipelines(/.*)?$                                                      /doc/start/data-management/pipelines$1",
  "^/doc/start/metrics-parameters-plots(/.*)?$                                            /doc/start/data-management/metrics-parameters-plots$1",
```

Reply:
> 302s because we're not sure how definitive this reorg is, so we can change the redirects in the future (instead of having to add additional redirects).